rosidl_generator_c/cmake/rosidl_generator_c_generate_interfaces.cmake

```diff
@@ -12,6 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+if(NOT DEFINED _ROSIDL_GENERATOR_C_GENERATE_INTERFACES)
+  set(_ROSIDL_GENERATOR_C_GENERATE_INTERFACES TRUE)
+else()
+  return()
+endif()
```

Comment:
This shouldn't be added since each registered extension point should only be included once.

Reply:
Done